pkgs/by-name/ph/phraze/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "phraze";
+  version = "0.3.10";
```

Comment:
New release dropped 2 hours ago if you'd like to combine it in this PR: https://github.com/sts10/phraze/releases/tag/v0.3.11